src/source/image_source.js

```diff
@@ -44,15 +44,32 @@ import type {
  *    ]
  * });
  *
- * // update
+ * // update coordinates
  * var mySource = map.getSource('some id');
  * mySource.setCoordinates([
  *     [-76.54335737228394, 39.18579907229748],
  *     [-76.52803659439087, 39.1838364847587],
  *     [-76.5295386314392, 39.17683392507606],
  *     [-76.54520273208618, 39.17876344106642]
  * ]);
- *
+ * 
+ * // update url. 
+ * // Note: this is most effective if your layer has the `raster-fade-duration` paint property set to 0.
+ * mySource.updateImage({
```

Comment:
We probably only need one example here. Just show it with `url` and `coordinates`. The docs should note that `coordinates` is optional.